src/trig.c

```diff
@@ -150,7 +23,18 @@ twin_fixed_t twin_sin(twin_angle_t a)
         /* mirror second and third quadrant values across y axis */
         if (a & TWIN_ANGLE_90)
             a = TWIN_ANGLE_180 - a;
-        sin = _twin_sin_table[a & (TWIN_ANGLE_90 - 1)];
+        /* 5th order polynomial approximation */
```

Comment:
Improve the comment, making it informative.

Reply:
@ndsl7109256
You can refer to the comments in [vwifi](https://github.com/sysprog21/vwifi/blob/main/vwifi.c), at least providing the form of the approximate polynomial and the description of the parameters.

```c
static inline s32 __sin_s3(s32 x)
{
...
    /* S(x) = (x * (3 * 2^p - (x * x)/2^r)) / 2^s
     * @n: the angle scale
     * @A: the amplitude
     * @p: keep the multiplication from overflowing
     */
...
}
```